packages/lit/src/lib/ColorToken/lib.ts

```diff
@@ -7,3 +9,97 @@ export function createHelperElement() {
   document.body.appendChild(element);
   return element;
 }
+
+const numberPattern = '(-?[\\d.]+(?:e[+-]?\\d+)?)';
+const colorComponentRegex = new RegExp(`${numberPattern}\\s+${numberPattern}\\s+${numberPattern}\\)$`);
+
+export function getCSSColorComponents(value: string) {
+  const [, a, b, c] = colorComponentRegex.exec(value)?.map(Number) || [NaN, NaN, NaN, NaN];
+  const components: [number, number, number] = [a, b, c];
+  return components;
+}
+
+export function limitColorComponents(
+  colorSpace: ColorSpace,
+  components: [number, number, number],
+): [number, number, number] {
```

Comment:
Dit zou misschien naar in de design-tokens-schema moeten uiteindelijk. Wat denk jij @bartveneman?